Formula/pdnsrec.rb

```diff
@@ -12,40 +12,37 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
-  depends_on :macos => :lion
+  depends_on "pkg-config" => :build
   depends_on "boost"
-  depends_on "lua" => :optional
+  depends_on "openssl"
+  depends_on "lua"
+  depends_on "gcc" => :linked if MacOS.version <= :mavericks
```

Comment:
>Sure, ideally.

It's test-bot that's rescuing `CompilerSelectionError` and then making a choice to install gcc as a remedy. If that behavior causes us to add `depends_on "gcc"` in the formula, it's the tail wagging the dog.